Topic-0

exequiel7 opened the pull request.

exequiel7 committed:
Add files via upload

exequiel7 committed:
first commit

galias11 commented:
- Esta faltando toda la parte de Layout y el Mock-up. Quizá te olvidaste de commitearlo.
- El form y el multimedia deben estar en archivos separados.
- Falta la TODO list.

exequiel7 committed:
mockup & my own layout

exequiel7 committed:
my todo list

exequiel7 commented:
> * Esta faltando toda la parte de Layout y el Mock-up. Quizá te olvidaste de commitearlo.
> * El form y el multimedia deben estar en archivos separados.
> * Falta la TODO list.

Answear: 
-> Cargue la parte de Layout, Mock-up y la TODO list.
-> Con respecto a el form y el multimedia, trate de interpretar el ejercicio y los fusione en un solo documento. Entiendo que tienen que estar separados, pero quizas entendí que la idea del ejercicio era comprender como colocar una imagen, un video y un audio y para no dejarlos aislados los coloque junto con el form.

galias11 reviewed (commented):
Te sigue faltando un ejercicio.
![image](https://user-images.githubusercontent.com/22265254/47510774-32bc0900-d84f-11e8-96f3-89d28d14a1ad.png)

Si tenes un rato revisalo, si me estoy equivocando yo y no lo veo por favor decime, pero me baje el branch y tampoco lo veo.

Saludos

exequiel7 commented:
Ahi lo vi, evidentemente lo pase de largo. Ahi lo hago y lo subo, perdon!

exequiel7 committed:
HTML & CSS basics

Perdon por la desprolijidad que maneje en el topic-0 a como subí los documentos.

galias11 commented on 00-HTML-CSS-basics/debug.log:
```diff
@@ -0,0 +1,3 @@
+[1012/131631.563:ERROR:crash_report_database_win.cc(428)] unexpected header
```
Este tipo de archivos no los incluyas en el commit

galias11 reviewed (approved):
Ahora si estan todos los ejercicios, te marque nomás una cosita. Los archivos .log no los subas en el pr (siempre agrega lo que sea relevante).

Te olvidaste de dos preguntas que había en los topics, pero no es necesario que vuelvas a subirlo. Si podes verlas para vos mejor, pero no es necesario que actualices el PR.
